Samples.Swift/Samples.Swift/API/PayPal.API.swift

```diff
@@ -5,69 +5,63 @@
 //  Created by Haider Khan on 5/17/20.
 //  Copyright © 2020 PayPal. All rights reserved.
 //
-
 import Foundation
+import PayPalCheckout
 
 class PayPal {
 
   // MARK: - Singleton
-
   static let shared = PayPal()
   private init() {}
 
   // MARK: - JSON
-
   static let jsonDecoder: JSONDecoder = {
     JSONDecoder()
   }()
+
   static let jsonEncoder: JSONEncoder = {
     let encoder = JSONEncoder()
     encoder.outputFormatting = .prettyPrinted
     return encoder
   }()
 
   // MARK: - Attributes
-
   static let clientId: String = "<client_id>"
+  static let returnUrl: String = "<redirect_uri>"
   private static let nodeAppBaseURL: String = "http://localhost:3000/"
   private static let baseURLv2: String = "https://api.sandbox.paypal.com/v2/"
 
   private var _accessToken: String?
 
   // MARK: - Errors
-
   enum Errors: String, Error {
     case noData
     case decodingError
   }
 
   // MARK: - API
-
   enum API {
 
     case fetchAccessToken(AccessTokenRequest)
-    case createOrder(CreateOrderRequest)
-    case captureOrder(CaptureOrderRequest)
+    case createOrder(OrderRequest)
 
     var path: String {
       switch self {
       case .fetchAccessToken:
         return PayPal.nodeAppBaseURL + "auth/token"
+
       case .createOrder:
         return PayPal.baseURLv2 + "checkout/orders"
-      case .captureOrder(let request):
-        return request.captureLink.href
       }
     }
 
     var method: String {
       switch self {
       case .fetchAccessToken:
         return "POST"
+
       case .createOrder:
         return "POST"
-      case .captureOrder(let request):
-        return request.captureLink.method
       }
     }
```

Comment:
Looks like this returns "POST" in all cases, so we can return that without switching on `self`